cylc/uiserver/schema.py

```diff
@@ -684,40 +767,63 @@ def run_jobs_query(
     # filter by task name (special UIS argument for namespace queries)
     if tasks:
         where_stmts.append(
-            r'(name = '
-            + r" OR name = ".join('?' for task in tasks)
+            r'(data.name = '
+            + r" OR data.name = ".join('?' for task in tasks)
```

Comment:
There are other `WHERE` conditions above, do these also need `data.` prepended to the variables in them?

Reply:
As above they aren't needed. But adding to keep the code consistent